File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/imagesearch/provider/bingpics/BingSearchConfig.java
package com.github.bumblebee.command.imagesearch.provider.bingpics;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Configuration
@ConfigurationProperties(prefix = "bing.api")
public class BingSearchConfig {

    private String accountKey;

    public String getAccountKey() {
        return accountKey;
    }

    public void setAccountKey(String accountKey) {
        this.accountKey = accountKey;
    }
}

File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/imagesearch/provider/bing/BingSearchConfig.kt
package com.github.bumblebee.command.imagesearch.provider.bing

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration

@Configuration
@ConfigurationProperties(prefix = "bing.api")
class BingSearchConfig{
    lateinit var accountKey: String
}
